governator-jetty/src/main/java/com/netflix/governator/guice/jetty/JettyModule.java

```diff
@@ -100,7 +105,14 @@ public JettyRunner(Server server, final LifecycleManager manager) {
             LOG.info("Jetty server starting");
             try {
                 server.start();
-                port = ((ServerConnector)server.getConnectors()[0]).getLocalPort();
+                int port = -1;
+                for (Connector connector : server.getConnectors()) {
+                    if (connector.getName().equals(PLAINTEXT_CONNECTOR_NAME)) {
+                        port = ((ServerConnector)connector).getLocalPort();
+                        break;
+                    }
+                }
+                this.port = port;
```

Comment:
what happens if no connector found?  should we throw an exception?

Reply:
Yea, good call. Adding an exception.